catalog/Maintenance_Monitoring/Health_Check.yml

```diff
@@ -0,0 +1,12 @@
+name: Health Check
+description:
+  projects:
```

Comment:
This indentation is wrong and needs to be top level, this currently makes the build fail